test/checkout_test.rb

```diff
@@ -0,0 +1,87 @@
+require "test_helper"
+
+class CheckoutTest < Test::Unit::TestCase
+  def setup
+    super
+    ShopifyAPI::VaultSession.format = ShopifyAPI::VaultSessionFormat
```

Comment:
Why is this necessary if you're already doing this in line 22 of `lib/shopify_api/resources/vault_session.rb`